docs/docsite/rst/user_guide/playbooks_variables.rst

```diff
@@ -835,8 +835,8 @@ Here is the order of precedence from least to greatest (the last listed variable
   * inventory group_vars/* [3]_
   * playbook group_vars/* [3]_
   * inventory file or script host vars [2]_
-  * inventory host_vars/*
-  * playbook host_vars/*
+  * inventory host_vars/all [3]_
+  * playbook host_vars/all [3]_
```

Comment:
this does the opposite of what the issue text implies, you are duplicating 'all' entries, which already exist at the lower precedence